{"id":1922,"date":"2022-11-18T01:13:36","date_gmt":"2022-11-18T01:13:36","guid":{"rendered":"http:\/\/example.com\/?p=925"},"modified":"2022-11-18T01:13:36","modified_gmt":"2022-11-18T01:13:36","slug":"925-umap-and-t-sne","status":"publish","type":"post","link":"https:\/\/w1.geleleger.nl\/rtgsk-925-umap-and-t-sne-unbrs\/","title":{"rendered":"Umap And T Sne"},"content":{"rendered":"<div class=\"a1420ecab7e2234e1f4e9be5cee323bb\" data-index=\"1\" style=\"float: none; margin:10px 0 10px 0; text-align:center;\">\n<script type=\"text\/javascript\">\r\n\tatOptions = {\r\n\t\t'key' : '4a1f1119f949a4af74d56b8a3af8b867',\r\n\t\t'format' : 'iframe',\r\n\t\t'height' : 250,\r\n\t\t'width' : 300,\r\n\t\t'params' : {}\r\n\t};\r\n\tdocument.write('<scr' + 'ipt type=\"text\/javascript\" src=\"http' + (location.protocol === 'https:' ? 's' : '') + ':\/\/bleedingofficecontagion.com\/4a1f1119f949a4af74d56b8a3af8b867\/invoke.js\"><\/scr' + 'ipt>');\r\n<\/script>\n<\/div>\n\n<p><strong>Umap And T Sne<\/strong>. <strong>In this work, we uncover their conceptual connection via a new insight into contrastive learning methods. As the number of data points increase, UMAP becomes more time efficient compared to TSNE. In this story, we are gonna go through three Dimensionality reduction techniques specifically used for Data Visualization: PCA, t-SNE, LDA and UMAP. What pitfalls should we consider when interpreting UMAP or t-SNE plot for single-cell RNA-Seq? Both algorithms minimize their loss functions by. The fundamental reasons for this and, more generally, the exact relationship between t -SNE and UMAP have remained unclear. At the core of both t-SNE and UMAP are loss functions that make similar points attract each other and push dissimilar points away from each other. The latter category, which includes SNE (Stochastic Neighbor Embedding), t-SNE (t-distributed Stochastic Neighbor Embedding), and UMAP (Uniform Manifold Approximation and Projection), will be the focus of this discussion.<\/strong><\/p>\n<p><figure> \t\t<img decoding=\"async\" src=\"https:\/\/community.jmp.com\/t5\/image\/serverpage\/image-id\/15862iFD97AD7DD06A292A?v=v2\" alt=\"Data visualization with t-SNE and UMAP - JMP User Community\" style=\"width: 100%; padding: 5px; background-color: grey;\"><figcaption>Data visualization with t-SNE and UMAP &#8211; JMP User Community (Georgia Rodriguez)<\/figcaption><\/figure>\n<p> \t \t<strong>Umap And T Sne<\/strong>. <strong>In this article, we&#39;ll give you the answers in the simplest manner possible. Indeed, both tSNE and UMAP were designed to predominantly preserve local structure that is to group neighboring data points together which indeed provides a very informative visualization of heterogeneity in your data. We are going to explore them in details using the Sign Language MNIST Dataset, without going in-depth with the maths behind the algorithms. In this work, we uncover their conceptual connection via a new insight into contrastive learning methods. Noise-contrastive estimation can be used to optimize t -SNE, while UMAP relies on negative sampling, another.<\/strong> Umap And T Sne. <\/p>\n<h3>At the core of both t-SNE and UMAP are loss functions that make similar points attract each other and push dissimilar points away from each other.<\/h3>\n<p>The latter category, which includes SNE (Stochastic Neighbor Embedding), t-SNE (t-distributed Stochastic Neighbor Embedding), and UMAP (Uniform Manifold Approximation and Projection), will be the focus of this discussion.<\/p>\n<div style=\"display: grid;grid-template-columns: auto auto auto; background: #eee; \">\n<div style=\"padding: 1em;\"> \t\t        <a href=\"https:\/\/bioconductor.org\/books\/3.13\/OSCA.advanced\/more-reddim_files\/figure-html\/densne-1.png\"><img decoding=\"async\" src=\"https:\/\/bioconductor.org\/books\/3.13\/OSCA.advanced\/more-reddim_files\/figure-html\/densne-1.png\" onerror=\"this.onerror=null;this.src='https:\/\/tse2.mm.bing.net\/th?id=OIP.wye7xfjqL0k0BaWQ-2SHLgHaEc&amp;pid=Api';\" style=\"width:100%;margin-top: 8px;vertical-align: middle;\"><\/a> \t\t        <\/p>\n<p>Chapter 4 Dimensionality reduction, redux | Advanced Single-Cell &#8230;<\/p>\n<\/p><\/div>\n<div style=\"padding: 1em;\"> \t\t        <a href=\"https:\/\/www.biaodianfu.com\/wp-content\/uploads\/2021\/07\/umap.png\"><img decoding=\"async\" src=\"https:\/\/www.biaodianfu.com\/wp-content\/uploads\/2021\/07\/umap.png\" onerror=\"this.onerror=null;this.src='https:\/\/tse3.mm.bing.net\/th?id=OIP.gtI_-smDvzmU1nqyNXpotAHaFj&amp;pid=Api';\" style=\"width:100%;margin-top: 8px;vertical-align: middle;\"><\/a> \t\t        <\/p>\n<p>\u6bd4t-SNE\u66f4\u597d\u7684\u964d\u7ef4\u7b97\u6cd5UMAP &#8211; \u6807\u70b9\u7b26<\/p>\n<\/p><\/div>\n<div style=\"padding: 1em;\"> \t\t        <a href=\"https:\/\/bioconductor.org\/books\/release\/OSCA.advanced\/more-reddim_files\/figure-html\/densne-1.png\"><img decoding=\"async\" src=\"https:\/\/bioconductor.org\/books\/release\/OSCA.advanced\/more-reddim_files\/figure-html\/densne-1.png\" onerror=\"this.onerror=null;this.src='https:\/\/tse1.mm.bing.net\/th?id=OIP.2_wIeCU7DYnvj9NGTq7XrAHaEc&amp;pid=Api';\" style=\"width:100%;margin-top: 8px;vertical-align: middle;\"><\/a> \t\t        <\/p>\n<p>Chapter 4 Dimensionality reduction, redux | Advanced Single-Cell &#8230;<\/p>\n<\/p><\/div>\n<\/p><\/div>\n<div style=\"display: grid;grid-template-columns: auto auto auto; background: #eee; \">\n<div style=\"padding: 1em;\"> \t\t        <a href=\"https:\/\/i.ytimg.com\/vi\/iE1WMWgrD9s\/maxresdefault.jpg\"><img decoding=\"async\" src=\"https:\/\/i.ytimg.com\/vi\/iE1WMWgrD9s\/maxresdefault.jpg\" onerror=\"this.onerror=null;this.src='https:\/\/tse4.mm.bing.net\/th?id=OIP.Ku6nF_K4_z9DbU_4kybGzQHaEK&amp;pid=Api';\" style=\"width:100%;margin-top: 8px;vertical-align: middle;\"><\/a> \t\t        <\/p>\n<p>Introduction to UMAP &#8211; YouTube<\/p>\n<\/p><\/div>\n<div style=\"padding: 1em;\"> \t\t        <a href=\"https:\/\/community.jmp.com\/t5\/image\/serverpage\/image-id\/17701iE843AE296DF51F65?v=v2\"><img decoding=\"async\" src=\"https:\/\/community.jmp.com\/t5\/image\/serverpage\/image-id\/17701iE843AE296DF51F65?v=v2\" onerror=\"this.onerror=null;this.src='https:\/\/tse1.mm.bing.net\/th?id=OIP.xJ2XZ2n98sx-DN6Mp8lPRwHaEo&amp;pid=Api';\" style=\"width:100%;margin-top: 8px;vertical-align: middle;\"><\/a> \t\t        <\/p>\n<p>Data visualization with t-SNE and UMAP &#8211; JMP User Community<\/p>\n<\/p><\/div>\n<div style=\"padding: 1em;\"> \t\t        <a href=\"https:\/\/miro.medium.com\/max\/1400\/1*gENT6BciUf2x3PkktMH-1g.png\"><img decoding=\"async\" src=\"https:\/\/miro.medium.com\/max\/1400\/1*gENT6BciUf2x3PkktMH-1g.png\" onerror=\"this.onerror=null;this.src='https:\/\/tse3.mm.bing.net\/th?id=OIP.4VcYlVpX7tAcZDsqlxOscgHaDt&amp;pid=Api';\" style=\"width:100%;margin-top: 8px;vertical-align: middle;\"><\/a> \t\t        <\/p>\n<p>tSNE vs. UMAP: Global Structure. Why preservation of global structure &#8230;<\/p>\n<\/p><\/div>\n<\/p><\/div>\n<div style=\"display: grid;grid-template-columns: auto auto auto; background: #eee; \">\n<div style=\"padding: 1em;\"> \t\t        <a href=\"https:\/\/img-blog.csdnimg.cn\/20210627105749703.png?x-oss-process=image\/watermark,type_ZmFuZ3poZW5naGVpdGk,shadow_10,text_aHR0cHM6Ly9ibG9nLmNzZG4ubmV0L3FxXzQzMzM3MjQ5,size_16,color_FFFFFF,t_70\"><img decoding=\"async\" src=\"https:\/\/img-blog.csdnimg.cn\/20210627105749703.png?x-oss-process=image\/watermark,type_ZmFuZ3poZW5naGVpdGk,shadow_10,text_aHR0cHM6Ly9ibG9nLmNzZG4ubmV0L3FxXzQzMzM3MjQ5,size_16,color_FFFFFF,t_70\" onerror=\"this.onerror=null;this.src='https:\/\/tse2.mm.bing.net\/th?id=OIP.Z48GzDmAV3LweU-HuMaW2AHaCV&amp;pid=Api';\" style=\"width:100%;margin-top: 8px;vertical-align: middle;\"><\/a> \t\t        <\/p>\n<p>\u5e38\u89c1\u7684PCA\u3001tSNE\u3001UMAP\u964d\u7ef4\u53ca\u805a\u7c7b\u57fa\u672c\u539f\u7406\u53ca\u4ee3\u7801\u5b9e\u4f8b | Coding\u6808<\/p>\n<\/p><\/div>\n<div style=\"padding: 1em;\"> \t\t        <a href=\"https:\/\/community.jmp.com\/t5\/image\/serverpage\/image-id\/19095i6C98CD51D910D49A?v=v2\"><img decoding=\"async\" src=\"https:\/\/community.jmp.com\/t5\/image\/serverpage\/image-id\/19095i6C98CD51D910D49A?v=v2\" onerror=\"this.onerror=null;this.src='https:\/\/tse2.mm.bing.net\/th?id=OIP.2Vj4ucdiN16GIBTwCcljdgHaDh&amp;pid=Api';\" style=\"width:100%;margin-top: 8px;vertical-align: middle;\"><\/a> \t\t        <\/p>\n<p>Data visualization with t-SNE and UMAP &#8211; Page 2 &#8211; JMP User Community<\/p>\n<\/p><\/div>\n<div style=\"padding: 1em;\"> \t\t        <a href=\"https:\/\/yuehhua.github.io\/slides\/bioinfo-tsne-umap\/pics\/2019.12.19.877522v1.full-3.png\"><img decoding=\"async\" src=\"https:\/\/yuehhua.github.io\/slides\/bioinfo-tsne-umap\/pics\/2019.12.19.877522v1.full-3.png\" onerror=\"this.onerror=null;this.src='https:\/\/tse1.mm.bing.net\/th?id=OIP.9PonunqYi3D8R7_jNTpq0wHaDt&amp;pid=Api';\" style=\"width:100%;margin-top: 8px;vertical-align: middle;\"><\/a> \t\t        <\/p>\n<p>bioinfo-tsne-umap slides<\/p>\n<\/p><\/div>\n<\/p><\/div>\n<div style=\"display: grid;grid-template-columns: auto auto auto; background: #eee; \">\n<div style=\"padding: 1em;\"> \t\t        <a href=\"https:\/\/www.researchgate.net\/publication\/354018342\/figure\/fig1\/AS:1058722117865472@1629430431382\/Interface-of-2D-visualization-via-UMAP-t-SNE-for-the-Mouse-Wang-2021-dataset-On-the-left.png\"><img decoding=\"async\" src=\"https:\/\/www.researchgate.net\/publication\/354018342\/figure\/fig1\/AS:1058722117865472@1629430431382\/Interface-of-2D-visualization-via-UMAP-t-SNE-for-the-Mouse-Wang-2021-dataset-On-the-left.png\" onerror=\"this.onerror=null;this.src='https:\/\/tse3.mm.bing.net\/th?id=OIP.Kb9R9PAZhJfE8MF5JD9YkwHaDn&amp;pid=Api';\" style=\"width:100%;margin-top: 8px;vertical-align: middle;\"><\/a> \t\t        <\/p>\n<p>Interface of 2D visualization via UMAP\/t-SNE for the Mouse-Wang-2021 &#8230;<\/p>\n<\/p><\/div>\n<div style=\"padding: 1em;\"> \t\t        <a href=\"https:\/\/media.springernature.com\/lw685\/springer-static\/image\/art:10.1038\/s41587-020-00809-z\/MediaObjects\/41587_2020_809_Fig1_HTML.png\"><img decoding=\"async\" src=\"https:\/\/media.springernature.com\/lw685\/springer-static\/image\/art:10.1038\/s41587-020-00809-z\/MediaObjects\/41587_2020_809_Fig1_HTML.png\" onerror=\"this.onerror=null;this.src='https:\/\/tse4.mm.bing.net\/th?id=OIP.7w5UHaA87-bK9ynisHOpTAHaBi&amp;pid=Api';\" style=\"width:100%;margin-top: 8px;vertical-align: middle;\"><\/a> \t\t        <\/p>\n<p>Initialization is critical for preserving global data structure in both &#8230;<\/p>\n<\/p><\/div>\n<div style=\"padding: 1em;\"> \t\t        <a href=\"https:\/\/ask.qcloudimg.com\/http-save\/7469656\/q6qeo1fwof.png\"><img decoding=\"async\" src=\"https:\/\/ask.qcloudimg.com\/http-save\/7469656\/q6qeo1fwof.png\" onerror=\"this.onerror=null;this.src='https:\/\/tse3.mm.bing.net\/th?id=OIP.uoRSFNOiXwdx5xM6FS3xOQHaFA&amp;pid=Api';\" style=\"width:100%;margin-top: 8px;vertical-align: middle;\"><\/a> \t\t        <\/p>\n<p>UMAP\u7684\u521d\u6b65\u4e86\u89e3\u53ca\u4e0et-SNE\u7684\u6bd4\u8f83-\u817e\u8baf\u4e91\u5f00\u53d1\u8005\u793e\u533a-\u817e\u8baf\u4e91<\/p>\n<\/p><\/div>\n<\/p><\/div>\n<p><strong>Umap And T Sne<\/strong>. <strong>We are going to explore them in details using the Sign Language MNIST Dataset, without going in-depth with the maths behind the algorithms. As the number of data points increase, UMAP becomes more time efficient compared to TSNE. However, one limitation with UMAP and t-SNE is that they can only capture the local structure of the data, the global structure of the data is not faithfully preserved. Noise-contrastive estimation can be used to optimize t -SNE, while UMAP relies on negative sampling, another. UMAP and t-SNE were ran independently on the dataset to qualitatively compare the distribution of clustered cell types in both reduced dimensions.<\/p>\n<p><\/strong> Umap And T Sne.<\/p>\n<div class=\"a1420ecab7e2234e1f4e9be5cee323bb\" data-index=\"2\" style=\"float: none; margin:10px 0 10px 0; text-align:center;\">\n<script type=\"text\/javascript\">\r\n\tatOptions = {\r\n\t\t'key' : '4a1f1119f949a4af74d56b8a3af8b867',\r\n\t\t'format' : 'iframe',\r\n\t\t'height' : 250,\r\n\t\t'width' : 300,\r\n\t\t'params' : {}\r\n\t};\r\n\tdocument.write('<scr' + 'ipt type=\"text\/javascript\" src=\"http' + (location.protocol === 'https:' ? 's' : '') + ':\/\/bleedingofficecontagion.com\/4a1f1119f949a4af74d56b8a3af8b867\/invoke.js\"><\/scr' + 'ipt>');\r\n<\/script>\n<\/div>\n\n<div style=\"font-size: 0px; height: 0px; line-height: 0px; margin: 0; padding: 0; clear: both;\"><\/div>","protected":false},"excerpt":{"rendered":"<p>Umap And T Sne. In this work, we uncover their conceptual connection via a new insight into contrastive learning methods. As the number of data points increase, UMAP becomes more time efficient compared to TSNE. In this story, we are gonna go through three Dimensionality reduction techniques specifically used for [&hellip;]<\/p>\n","protected":false},"author":1,"featured_media":0,"comment_status":"open","ping_status":"closed","sticky":false,"template":"","format":"standard","meta":{"footnotes":""},"categories":[1],"tags":[],"aioseo_notices":[],"_links":{"self":[{"href":"https:\/\/w1.geleleger.nl\/wp-json\/wp\/v2\/posts\/1922"}],"collection":[{"href":"https:\/\/w1.geleleger.nl\/wp-json\/wp\/v2\/posts"}],"about":[{"href":"https:\/\/w1.geleleger.nl\/wp-json\/wp\/v2\/types\/post"}],"author":[{"embeddable":true,"href":"https:\/\/w1.geleleger.nl\/wp-json\/wp\/v2\/users\/1"}],"replies":[{"embeddable":true,"href":"https:\/\/w1.geleleger.nl\/wp-json\/wp\/v2\/comments?post=1922"}],"version-history":[{"count":0,"href":"https:\/\/w1.geleleger.nl\/wp-json\/wp\/v2\/posts\/1922\/revisions"}],"wp:attachment":[{"href":"https:\/\/w1.geleleger.nl\/wp-json\/wp\/v2\/media?parent=1922"}],"wp:term":[{"taxonomy":"category","embeddable":true,"href":"https:\/\/w1.geleleger.nl\/wp-json\/wp\/v2\/categories?post=1922"},{"taxonomy":"post_tag","embeddable":true,"href":"https:\/\/w1.geleleger.nl\/wp-json\/wp\/v2\/tags?post=1922"}],"curies":[{"name":"wp","href":"https:\/\/api.w.org\/{rel}","templated":true}]}}