{"id":1914,"date":"2022-10-18T01:53:57","date_gmt":"2022-10-18T01:53:57","guid":{"rendered":"http:\/\/example.com\/?p=917"},"modified":"2022-10-18T01:53:57","modified_gmt":"2022-10-18T01:53:57","slug":"917-umap-analysis","status":"publish","type":"post","link":"https:\/\/w1.geleleger.nl\/rtgsk-917-umap-analysis-unbrs\/","title":{"rendered":"Umap Analysis"},"content":{"rendered":"<div class=\"a1420ecab7e2234e1f4e9be5cee323bb\" data-index=\"1\" style=\"float: none; margin:10px 0 10px 0; text-align:center;\">\n<script type=\"text\/javascript\">\r\n\tatOptions = {\r\n\t\t'key' : '4a1f1119f949a4af74d56b8a3af8b867',\r\n\t\t'format' : 'iframe',\r\n\t\t'height' : 250,\r\n\t\t'width' : 300,\r\n\t\t'params' : {}\r\n\t};\r\n\tdocument.write('<scr' + 'ipt type=\"text\/javascript\" src=\"http' + (location.protocol === 'https:' ? 's' : '') + ':\/\/bleedingofficecontagion.com\/4a1f1119f949a4af74d56b8a3af8b867\/invoke.js\"><\/scr' + 'ipt>');\r\n<\/script>\n<\/div>\n\n<p><strong>Umap Analysis<\/strong>. <strong>UMAP (Uniform Manifold Approximation and Projection) is a novel manifold learning technique for dimension reduction. It is similar to PCA (Principal Component Analysis) in terms of speed and resembles tSNE to reduce dimensionality while preserving as much information of the dataset as possible. This article will discuss how the algorithm works in practice. The algorithm is founded on three assumptions about the data The data is uniformly distributed on Riemannian manifold; UMAP is an algorithm for dimension reduction based on manifold learning techniques and ideas from topological data analysis. Uniform Manifold Approximation and Projection (UMAP) is a dimension reduction technique that can be used for visualisation similarly to t-SNE, but also for general non-linear dimension reduction. It provides a very general framework for approaching manifold learning and dimension reduction, but can also provide specific concrete realizations. UMAP is a dimensionality reduction algorithm and a powerful data analysis tool. UMAP is a general purpose manifold learning and dimension reduction algorithm.<\/strong><\/p>\n<p><figure> \t\t<img decoding=\"async\" src=\"https:\/\/www.researchgate.net\/publication\/338649913\/figure\/fig16\/AS:859602853830656@1581956699044\/Fig-S14-Diffusion-map-and-UMAP-analysis-of-mouse-small-intestine-related-to-Figure-3.png\" alt=\"Fig. S14. Diffusion map and UMAP analysis of mouse small intestine ...\" style=\"width: 100%; padding: 5px; background-color: grey;\"><figcaption>Fig. S14. Diffusion map and UMAP analysis of mouse small intestine &#8230; (Floyd Bates)<\/figcaption><\/figure>\n<p> \t \t<strong>Umap Analysis<\/strong>. <strong>Here we apply uniform manifold approximation and projection (UMAP), a non-linear dimension reduction. UMAP is constructed from a theoretical framework based in Riemannian geometry and algebraic topology. It is similar to PCA (Principal Component Analysis) in terms of speed and resembles tSNE to reduce dimensionality while preserving as much information of the dataset as possible. Uniform Manifold Approximation and Projection (or UMAP) is a new dimension reduction technique that can be used to visualize patterns of clustering in high-dimensional data. Whereas PCoA performs an eigendecomposition that focuses on linearly preserving the pairwise distances between the samples (global structure), UMAP uses a nonlinear graph construction and embedding method to optimize an.<\/strong> Umap Analysis. <\/p>\n<h3>UMAP embedding of the scRNA-seq dataset containing dorsal and ventral.<\/h3>\n<p>The following UMAP is my result: And this is the published UMAP: Since the dataset contain smoker and non-smoker data, I played around with the merging sequence, i.e.: merging the data sequentially based on their IDs vs. merging the same type of data first (smoker with smoker and non-smoker with non-smoker) before merging all of them together.<\/p>\n<div style=\"display: grid;grid-template-columns: auto auto auto; background: #eee; \">\n<div style=\"padding: 1em;\"> \t\t        <a href=\"https:\/\/www.researchgate.net\/publication\/350096336\/figure\/fig1\/AS:1080293037473824@1634573339191\/scRNA-seq-analysis-of-CD45-cells-in-healthy-kidney-a-UMAP-plot-shows-clustering_Q320.jpg\"><img decoding=\"async\" src=\"https:\/\/www.researchgate.net\/publication\/350096336\/figure\/fig1\/AS:1080293037473824@1634573339191\/scRNA-seq-analysis-of-CD45-cells-in-healthy-kidney-a-UMAP-plot-shows-clustering_Q320.jpg\" onerror=\"this.onerror=null;this.src='https:\/\/tse2.mm.bing.net\/th?id=OIP.LZzWQQRY1msstu5gk2gSxQAAAA&amp;pid=Api';\" style=\"width:100%;margin-top: 8px;vertical-align: middle;\"><\/a> \t\t        <\/p>\n<p>scRNA-seq analysis of CD45+ cells in healthy kidney. (a) UMAP plot &#8230;<\/p>\n<\/p><\/div>\n<div style=\"padding: 1em;\"> \t\t        <a href=\"https:\/\/www.researchgate.net\/publication\/343355592\/figure\/fig2\/AS:926724002959362@1597959628515\/Single-cell-RNA-sequencing-analysis-of-midbrain-specific-organoids-a-Uniform-manifold.png\"><img decoding=\"async\" src=\"https:\/\/www.researchgate.net\/publication\/343355592\/figure\/fig2\/AS:926724002959362@1597959628515\/Single-cell-RNA-sequencing-analysis-of-midbrain-specific-organoids-a-Uniform-manifold.png\" onerror=\"this.onerror=null;this.src='https:\/\/tse2.mm.bing.net\/th?id=OIP.FZhWzTdF6WckU7Mm5ZwCMAHaF3&amp;pid=Api';\" style=\"width:100%;margin-top: 8px;vertical-align: middle;\"><\/a> \t\t        <\/p>\n<p>Single-cell RNA sequencing analysis of midbrain-specific organoids. (a &#8230;<\/p>\n<\/p><\/div>\n<div style=\"padding: 1em;\"> \t\t        <a href=\"https:\/\/www.researchgate.net\/publication\/339909076\/figure\/fig1\/AS:941331211878407@1601442258266\/Single-cell-RNA-sequencing-analysis-reveals-cellular-heterogeneity-of-SPC-including.png\"><img decoding=\"async\" src=\"https:\/\/www.researchgate.net\/publication\/339909076\/figure\/fig1\/AS:941331211878407@1601442258266\/Single-cell-RNA-sequencing-analysis-reveals-cellular-heterogeneity-of-SPC-including.png\" onerror=\"this.onerror=null;this.src='https:\/\/tse3.mm.bing.net\/th?id=OIP.6eDrjTk3qZlLO0nCaM1KvQHaJD&amp;pid=Api';\" style=\"width:100%;margin-top: 8px;vertical-align: middle;\"><\/a> \t\t        <\/p>\n<p>Single-cell RNA-sequencing analysis reveals cellular heterogeneity of &#8230;<\/p>\n<\/p><\/div>\n<\/p><\/div>\n<div style=\"display: grid;grid-template-columns: auto auto auto; background: #eee; \">\n<div style=\"padding: 1em;\"> \t\t        <a href=\"https:\/\/www.researchgate.net\/profile\/Jonas_Schupp2\/publication\/340810066\/figure\/fig5\/AS:959163526025259@1605693813775\/ScRNA-seq-identifies-CTHRC1-pathologic-fibroblasts-in-human-fibrotic-lungs-a-UMAP-plot.png\"><img decoding=\"async\" src=\"https:\/\/www.researchgate.net\/profile\/Jonas_Schupp2\/publication\/340810066\/figure\/fig5\/AS:959163526025259@1605693813775\/ScRNA-seq-identifies-CTHRC1-pathologic-fibroblasts-in-human-fibrotic-lungs-a-UMAP-plot.png\" onerror=\"this.onerror=null;this.src='https:\/\/tse3.mm.bing.net\/th?id=OIP.pg5kYLeOObrTWV22DUviggHaGp&amp;pid=Api';\" style=\"width:100%;margin-top: 8px;vertical-align: middle;\"><\/a> \t\t        <\/p>\n<p>ScRNA-seq identifies CTHRC1\u207a pathologic fibroblasts in human fibrotic &#8230;<\/p>\n<\/p><\/div>\n<div style=\"padding: 1em;\"> \t\t        <a href=\"https:\/\/www.researchgate.net\/publication\/358783385\/figure\/download\/fig6\/AS:1126322336215040@1645547579333\/Single-cell-sequencing-analysis-A-UMAP-plot-showing-8-cell-clusters-Gray-area.png\"><img decoding=\"async\" src=\"https:\/\/www.researchgate.net\/publication\/358783385\/figure\/download\/fig6\/AS:1126322336215040@1645547579333\/Single-cell-sequencing-analysis-A-UMAP-plot-showing-8-cell-clusters-Gray-area.png\" onerror=\"this.onerror=null;this.src='https:\/\/tse2.mm.bing.net\/th?id=OIP.PRTQoV18wcJG4Tzh6vKC1wHaIg&amp;pid=Api';\" style=\"width:100%;margin-top: 8px;vertical-align: middle;\"><\/a> \t\t        <\/p>\n<p>| Single cell sequencing analysis (A). UMAP plot showing 8 cell &#8230;<\/p>\n<\/p><\/div>\n<div style=\"padding: 1em;\"> \t\t        <a href=\"https:\/\/e-learn.csc.fi\/pluginfile.php\/478\/course\/section\/99\/UMAP%20plot.png\"><img decoding=\"async\" src=\"https:\/\/e-learn.csc.fi\/pluginfile.php\/478\/course\/section\/99\/UMAP%20plot.png\" onerror=\"this.onerror=null;this.src='https:\/\/tse1.mm.bing.net\/th?id=OIP.kZ4Z_wRnYUdTcn9xcweZNgHaHR&amp;pid=Api';\" style=\"width:100%;margin-top: 8px;vertical-align: middle;\"><\/a> \t\t        <\/p>\n<p>Course: Single cell RNA-seq data analysis with Chipster<\/p>\n<\/p><\/div>\n<\/p><\/div>\n<div style=\"display: grid;grid-template-columns: auto auto auto; background: #eee; \">\n<div style=\"padding: 1em;\"> \t\t        <a href=\"https:\/\/www.researchgate.net\/publication\/354161349\/figure\/fig5\/AS:1061286259478528@1630041770926\/Comparative-single-cell-RNA-sequencing-analysis-of-EpiTS-embryoids-a-UMAP-plots-showing.png\"><img decoding=\"async\" src=\"https:\/\/www.researchgate.net\/publication\/354161349\/figure\/fig5\/AS:1061286259478528@1630041770926\/Comparative-single-cell-RNA-sequencing-analysis-of-EpiTS-embryoids-a-UMAP-plots-showing.png\" onerror=\"this.onerror=null;this.src='https:\/\/tse4.mm.bing.net\/th?id=OIP.7f9sxOpU-jazAAUY9EfpzwHaIY&amp;pid=Api';\" style=\"width:100%;margin-top: 8px;vertical-align: middle;\"><\/a> \t\t        <\/p>\n<p>Comparative single-cell RNA-sequencing analysis of EpiTS embryoids a &#8230;<\/p>\n<\/p><\/div>\n<div style=\"padding: 1em;\"> \t\t        <a href=\"https:\/\/bustools.github.io\/BUS_notebooks_R\/figure\/slingshot.Rmd\/umap_clust-1.png\"><img decoding=\"async\" src=\"https:\/\/bustools.github.io\/BUS_notebooks_R\/figure\/slingshot.Rmd\/umap_clust-1.png\" onerror=\"this.onerror=null;this.src='https:\/\/tse3.mm.bing.net\/th?id=OIP.MUfXp9f-DcIcd61wI-UXCAHaFS&amp;pid=Api';\" style=\"width:100%;margin-top: 8px;vertical-align: middle;\"><\/a> \t\t        <\/p>\n<p>Pseudotime analysis with slingshot<\/p>\n<\/p><\/div>\n<div style=\"padding: 1em;\"> \t\t        <a href=\"https:\/\/www.researchgate.net\/publication\/331238431\/figure\/fig3\/AS:731591684943892@1551436458522\/Lineage-progression-a-UMAP-plot-as-shown-in-Fig1c-coloured-by-the-density-of-each-cell.jpg\"><img decoding=\"async\" src=\"https:\/\/www.researchgate.net\/publication\/331238431\/figure\/fig3\/AS:731591684943892@1551436458522\/Lineage-progression-a-UMAP-plot-as-shown-in-Fig1c-coloured-by-the-density-of-each-cell.jpg\" onerror=\"this.onerror=null;this.src='https:\/\/tse3.mm.bing.net\/th?id=OIP.YzCJYQl1faY4dxwcs-CD6AHaHJ&amp;pid=Api';\" style=\"width:100%;margin-top: 8px;vertical-align: middle;\"><\/a> \t\t        <\/p>\n<p>Lineage progression a, UMAP plot as shown in Fig. 1c, coloured by the &#8230;<\/p>\n<\/p><\/div>\n<\/p><\/div>\n<div style=\"display: grid;grid-template-columns: auto auto auto; background: #eee; \">\n<div style=\"padding: 1em;\"> \t\t        <a href=\"https:\/\/umap-learn.readthedocs.io\/en\/latest\/_images\/parameters_8_1.png\"><img decoding=\"async\" src=\"https:\/\/umap-learn.readthedocs.io\/en\/latest\/_images\/parameters_8_1.png\" onerror=\"this.onerror=null;this.src='https:\/\/tse4.mm.bing.net\/th?id=OIP.CEdgd9u09F8D8zXVrJm8NQHaFZ&amp;pid=Api';\" style=\"width:100%;margin-top: 8px;vertical-align: middle;\"><\/a> \t\t        <\/p>\n<p>Basic UMAP Parameters \u2014 umap 0.5 documentation<\/p>\n<\/p><\/div>\n<div style=\"padding: 1em;\"> \t\t        <a href=\"https:\/\/www.researchgate.net\/publication\/344144904\/figure\/fig5\/AS:938433811845121@1600751464276\/UMAP-representation-of-dimensionally-reduced-data-following-graph-based-clustering.ppm\"><img decoding=\"async\" src=\"https:\/\/www.researchgate.net\/publication\/344144904\/figure\/fig5\/AS:938433811845121@1600751464276\/UMAP-representation-of-dimensionally-reduced-data-following-graph-based-clustering.ppm\" onerror=\"this.onerror=null;this.src='https:\/\/tse4.mm.bing.net\/th?id=OIP.NoIwxg9pP8OhWdRvHm0NDwHaKP&amp;pid=Api';\" style=\"width:100%;margin-top: 8px;vertical-align: middle;\"><\/a> \t\t        <\/p>\n<p>UMAP representation of dimensionally reduced data following graph-based &#8230;<\/p>\n<\/p><\/div>\n<div style=\"padding: 1em;\"> \t\t        <a href=\"https:\/\/images.squarespace-cdn.com\/content\/v1\/5bc760564d546e3f72b825c1\/1558631363684-CVG1SEH4D3P37TQWNLQJ\/ke17ZwdGBToddI8pDm48kDv2giyDE94CmqtrXZjtRVkUqsxRUqqbr1mOJYKfIPR7LoDQ9mXPOjoJoqy81S2I8N_N4V1vUb5AoIIIbLZhVYy7Mythp_T-mtop-vrsUOmeInPi9iDjx9w8K4ZfjXt2dkwHTmniiTbZDYqDeTiA4CPXmphBez8WDQkqU1amEGa1CjLISwBs8eEdxAxTptZAUg\/Screen+Shot+2019-05-23+at+11.09.07+AM.png\"><img decoding=\"async\" src=\"https:\/\/images.squarespace-cdn.com\/content\/v1\/5bc760564d546e3f72b825c1\/1558631363684-CVG1SEH4D3P37TQWNLQJ\/ke17ZwdGBToddI8pDm48kDv2giyDE94CmqtrXZjtRVkUqsxRUqqbr1mOJYKfIPR7LoDQ9mXPOjoJoqy81S2I8N_N4V1vUb5AoIIIbLZhVYy7Mythp_T-mtop-vrsUOmeInPi9iDjx9w8K4ZfjXt2dkwHTmniiTbZDYqDeTiA4CPXmphBez8WDQkqU1amEGa1CjLISwBs8eEdxAxTptZAUg\/Screen+Shot+2019-05-23+at+11.09.07+AM.png\" onerror=\"this.onerror=null;this.src='https:\/\/tse4.mm.bing.net\/th?id=OIP.WzSQQrTVvsllH_xh3OKCwgHaDj&amp;pid=Api';\" style=\"width:100%;margin-top: 8px;vertical-align: middle;\"><\/a> \t\t        <\/p>\n<p>The UMAP Plugin in FlowJo: A User&#39;s Review \u2014 The Clambey Lab<\/p>\n<\/p><\/div>\n<\/p><\/div>\n<p><strong>Umap Analysis<\/strong>. <strong>Its details are described by McInnes, Healy, and Melville and its official implementation is available through a python package umap-learn. UMAP is constructed from a theoretical framework based in Riemannian geometry and algebraic topology. Here we apply uniform manifold approximation and projection (UMAP), a non-linear dimension reduction. Larger values result in more global views of the manifold. Compute the n_neighbors nearest points for each data point in X under metric.<\/p>\n<p><\/strong> Umap Analysis.<\/p>\n<div class=\"a1420ecab7e2234e1f4e9be5cee323bb\" data-index=\"2\" style=\"float: none; margin:10px 0 10px 0; text-align:center;\">\n<script type=\"text\/javascript\">\r\n\tatOptions = {\r\n\t\t'key' : '4a1f1119f949a4af74d56b8a3af8b867',\r\n\t\t'format' : 'iframe',\r\n\t\t'height' : 250,\r\n\t\t'width' : 300,\r\n\t\t'params' : {}\r\n\t};\r\n\tdocument.write('<scr' + 'ipt type=\"text\/javascript\" src=\"http' + (location.protocol === 'https:' ? 's' : '') + ':\/\/bleedingofficecontagion.com\/4a1f1119f949a4af74d56b8a3af8b867\/invoke.js\"><\/scr' + 'ipt>');\r\n<\/script>\n<\/div>\n\n<div style=\"font-size: 0px; height: 0px; line-height: 0px; margin: 0; padding: 0; clear: both;\"><\/div>","protected":false},"excerpt":{"rendered":"<p>Umap Analysis. UMAP (Uniform Manifold Approximation and Projection) is a novel manifold learning technique for dimension reduction. It is similar to PCA (Principal Component Analysis) in terms of speed and resembles tSNE to reduce dimensionality while preserving as much information of the dataset as possible. This article will discuss how [&hellip;]<\/p>\n","protected":false},"author":1,"featured_media":0,"comment_status":"open","ping_status":"closed","sticky":false,"template":"","format":"standard","meta":{"footnotes":""},"categories":[1],"tags":[],"aioseo_notices":[],"_links":{"self":[{"href":"https:\/\/w1.geleleger.nl\/wp-json\/wp\/v2\/posts\/1914"}],"collection":[{"href":"https:\/\/w1.geleleger.nl\/wp-json\/wp\/v2\/posts"}],"about":[{"href":"https:\/\/w1.geleleger.nl\/wp-json\/wp\/v2\/types\/post"}],"author":[{"embeddable":true,"href":"https:\/\/w1.geleleger.nl\/wp-json\/wp\/v2\/users\/1"}],"replies":[{"embeddable":true,"href":"https:\/\/w1.geleleger.nl\/wp-json\/wp\/v2\/comments?post=1914"}],"version-history":[{"count":0,"href":"https:\/\/w1.geleleger.nl\/wp-json\/wp\/v2\/posts\/1914\/revisions"}],"wp:attachment":[{"href":"https:\/\/w1.geleleger.nl\/wp-json\/wp\/v2\/media?parent=1914"}],"wp:term":[{"taxonomy":"category","embeddable":true,"href":"https:\/\/w1.geleleger.nl\/wp-json\/wp\/v2\/categories?post=1914"},{"taxonomy":"post_tag","embeddable":true,"href":"https:\/\/w1.geleleger.nl\/wp-json\/wp\/v2\/tags?post=1914"}],"curies":[{"name":"wp","href":"https:\/\/api.w.org\/{rel}","templated":true}]}}